docs/content/users/extend/additional-services.md

```diff
@@ -144,7 +160,7 @@ The `install.yaml` is a simple YAML file with a few main sections:
     ```
 
 * `dependencies`: an array of add-ons that this add-on depends on.
-* `post_install_actions`: an array of Bash statements or scripts to be executed after `project_files` and `global_files` are installed. The actions are executed in the context of the target project’s `.ddev` directory.
+* `post_install_actions`: an array of Bash statements or scripts to be executed after `project_files` and `global_files` are installed. The actions are executed in the context of the target project’s `.ddev` directory. With DDEV v1.23.5+, variables from the `.ddev/.env.name` file, where `name` is the name of the add-on, are available in this context.
```

Comment:
So here, the .env.name is for the add-on name, true? But elsewhere, service name?

Reply:
Yes.